app/services/to_cocina/work/description_mapper.rb

```diff
@@ -26,11 +26,14 @@ def call
 
       attr_reader :work_form
 
+      delegate :publication_date, to: :work_form
```

Comment:
Why delegate this method to the work form but not others referenced in this class? Wondering if this inconsistency will cause confusion. At least comment-worthy?

Reply:
The reasons for which I did it do not justify the inconsistency, so removing delegation.